docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -227,87 +227,145 @@ sudo -E printenv
 
 ### How can I send a custom Email?
 
-Appcircle provides a ready-to-use email structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the three modules. If desired, the user can customize this structure by using the custom script below to send their own custom email.
+Appcircle provides a **ready-to-use email** structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the three modules. If desired, the user can customize this structure by using the custom script below to send their own custom email.
+
+The following script is set to use a [**Gmail SMTP Server**]. For more information, please visit [**Gmail SMTP Server**](https://support.google.com/a/answer/176600?hl=en) documentation. 
 
 ```bash
 
-os=""
-if uname -a | grep -iq "darwin"
-then
-	os="darwin"
-elif uname -a | grep -iq "linux"
-then
-	os="linux"
-fi
+# Set SMTP server 
+HOST_="smtp.gmail.com"
+PORT_="587"
+ACCOUNT_="gmail"
 
-if [ "$os" == "" ]
-then
-	echo "This script expects darwin or linux."
-	exit 1
-elif [ "$os" == "darwin" ] && which brew | grep -c "not found"
-then
-	echo "Can't find brew installation; make brew command visible or install homebrew and try again"
-	exit 1
-elif [ "$os" == "linux" ] && ! dpkg -s apt | grep -c "install ok installed"
-then
-	echo "apt is not installed; install apt and try again"
-	exit 1
-fi
+# Add account details (make sure to replace EMAIL_, USERNAME_, and PASSWORD_ with your actual details)
+EMAIL_="your-email-address@gmail.com"
+USERNAME_="your-email-address@gmail.com"
+PASSWORD_="your-email-password"
 
-if [ "$os" == "linux" ]
-then
-	sudo apt-get install mailutils
-	sudo apt-get install msmtp
-	sudo apt-get install msmtp-mta
+# Set e-mail details
+EMAIL_SUBJECT="Test Email Subject"
+EMAIL_TO="recieve-email-address"
+# This part will be used for visualization, Example usage: "Appcircle <test@gmail.com>"
+EMAIL_FROM="sender-email-address"
+EMAIL_BODY="This is the body of the test email."
+
+# Detect operating system
+os=""
+if uname -a | grep -iq "darwin"; then
+    os="darwin"
+elif uname -a | grep -iq "linux"; then
+    os="linux"
 fi
 
-if [ "$os" == "darwin" ]
-then
-	brew install mailutils
-	brew install msmtp
-	echo "set sendmail=/usr/local/bin/msmtp" | sudo tee -a /etc/mail.rc
+# Check if OS is supported and install necessary packages
+if [ "$os" == "darwin" ]; then
+    if ! command -v brew > /dev/null 2>&1; then
+        echo "Error msg: brew not found."
+        exit 1
+    fi
+    brew install mailutils msmtp
+    echo "set sendmail=/usr/local/bin/msmtp" | sudo tee -a /etc/mail.rc
+    echo "tls_fingerprint" >> ~/.msmtprc
```

Comment:
The code in the example I gave is pseudocode, that is, code that is not real to explain the algorithm. You shouldn't have taken it directly. Can you please change it back to its old code ?

```suggestion
    { echo -n "tls_fingerprint " && msmtp --serverinfo --tls --tls-certcheck=off --host=$HOST_ --port=$PORT_ | egrep -o "([0-9A-Za-z]{2}:){31}[0-9A-Za-z]{2}"; } >> ~/.msmtprc
```